contracts/ICS20Transfer.sol

```diff
@@ -321,6 +324,14 @@ contract ICS20Transfer is
         }
     }
 
+    /// @inheritdoc UUPSUpgradeable
+    function _authorizeUpgrade(address) internal virtual override {
```

Comment:
To avoid having this logic in both here and in IBCUUPSUpgradable, maybe we could have a method on IBCUUPSUpgradable so that you could just do something like `ics26Router.isAuthorizedToUpgrade`?

Reply:
I thought about this, since this function won't get called much, I didn't think this efficiency mattered. Do you feel strongly about this?

Reply:
Not so much about effeciency. The reason I think it might make sense is because we want to be sure that logic is the same. And not update only one of them by mistake or something.

Reply:
I did what you asked. Please check.

Reply:
Yeah, looks good to me!